test/unit/modules/blocks/verify.js

```diff
@@ -1931,6 +1937,44 @@ describe('blocks/verify', () => {
 			done();
 		});
 
+		describe('system headers', () => {
```

Comment:
This file is for unit tests of `modules/blocks/verify.js`, you shouldn't add describe block for `system headers`, but add corresponding paths in `processBlock` tests.

Reply:
It's consistent with the current suite structure.